src/typescript/frontend/tailwind.config.js

```diff
@@ -74,11 +74,16 @@ module.exports = {
         "0%": { transform: "translateX(0)" },
         "100%": { transform: "translateX(-4527.83px)" },
       },
+      banCarousel: {
+        "0%": { transform: "translateX(0)" },
+        "100%": { transform: "translateX(-5008.25px)" },
```

Comment:
I see you figured out the ways of the infinite carousel too...(this was such a pain lol, had to measure it so it wouldn't snap at the end of the animation)